static/app/utils/oxfordizeArray.spec.tsx

```diff
@@ -18,14 +18,22 @@ describe('oxfordizeArray', function () {
   });
 });
 
-describe('oxfordizeElements', function () {
+describe('Oxfordize', function () {
   it('correctly formats lists of elements', function () {
     const items = [<i key="1">one</i>, <i key="2">two</i>, <i key="3">three</i>];
-    render(oxfordizeElements(items));
+    render(<Oxfordize>{items}</Oxfordize>);
 
     expect(screen.getByText('one')).toBeInTheDocument();
     expect(screen.getByText('two')).toBeInTheDocument();
     expect(screen.getByText('three')).toBeInTheDocument();
     expect(screen.getByText(/, and/)).toBeInTheDocument();
   });
+
+  it('correctly formats mixed lists of nodes', function () {
+    const items = [<i key="1">one</i>, 'two'];
+    render(<Oxfordize>{items}</Oxfordize>);
+
+    expect(screen.getByText('one')).toBeInTheDocument();
+    expect(screen.getByText(/, and two/)).toBeInTheDocument();
```

Comment:
Isn't this different behavior than the non-element version where it's `A and B`?

Reply:
Indeed, the implementation in getsentry was wrong, I'll fix that up.